public/data/heroes.json

```diff
@@ -0,0 +1,7558 @@
+{
+	"Classes": [
```

Comment:
this could be an object where each `Class` value is a key instead of having to do lookups on objects in the array with `find` everytime. this will make your life a bit easier when trying to refer to a class in code (ie removes one full list lookup every time).